### PR TITLE
Add Capture feature to README

### PR DESCRIPTION
**Pull request by jmatsu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +90,43 @@ DeployGate.registerCallback(new DeployGateCallback() {
...
+Workspaces, which have activated the Capture feature, can storee screenshots, device states and Logcat at DeployGate. The Capture feature is enabled by default.
+
+```java
+import com.deploygate.sdk.DeployGate;
+import com.deploygate.sdk.DeployGateSdkConfiguration;
+
+DeployGateSdkConfiguration config =
+                new DeployGateSdkConfiguration().
+                   setCaptureEnabled(true). // Defaults to true
+                   build();
```
Comment (satsukies) [inline README.md:104]: We need to use the Builder class for making a configuration with builder pattern.

```suggestion
DeployGateSdkConfiguration config = 
    new DeployGateSdkConfiguration.Builder()
        .setCaptureEnabled(true) // Defaults to true
        .build();
```

File: README.md [hunk 2]
```diff
@@ -49,7 +49,14 @@ And also, you need to call `DeployGate#install` in your Application class, Conte
...
 
 ```java
-DeployGate.install(context, /** forceApplyOnReleaseBuild */ false);
+import com.deploygate.sdk.DeployGate;
+import com.deploygate.sdk.DeployGateSdkConfiguration;
+
+DeployGateSdkConfiguration config =
+                new DeployGateSdkConfiguration().
+                   setEnabledOnNonDebuggableBuild(true). // Make sure you set true here.
+                   build();
```
Comment (satsukies) [inline README.md:58]: ditto

```suggestion
DeployGateSdkConfiguration config =
    new DeployGateSdkConfiguration.Builder().
        setEnabledOnNonDebuggableBuild(true). // Make sure you set true here.
        build();
```

**Review by satsukies — CHANGES_REQUESTED** [review]
Thank you for the update!
It looks almost good to me.
Please check my suggestions.

File: README.md (hunk 1 above)
Comment (jmatsu) [reply README.md:104]: Oh, good catch

**Review by satsukies — APPROVED** [review]
LGTM 👍

**Comment by jmatsu** [comment]
Thanks~

**Pull request merged**